src/routes/docs/tooling/command-line/buckets/+page.markdoc

```diff
@@ -0,0 +1,119 @@
+---
+layout: article
+title: Deploy buckets with CLI
+description: Efficiently deploy your Appwrite buckets using the Command-Line Tool (CLI). 
+---
+
+{% info title="Before proceeding" %}
+Ensure you [**install**](/docs/tooling/command-line/installation#Installation) the CLI, [**login**](/docs/tooling/command-line/installation#get-started) to your Appwrite account, and [**initialize**](/docs/tooling/command-line/installation#Initialization) your Appwrite project.
+{% /info %}
+
+The Appwrite CLI allows you to configure and deploy buckets across projects.
```

Comment:
> across projects

Does it? 🧐

Reply:
Yes, I tested this myself to double check. Steps to reproduce:
1. install appwrite cli
2. log in to your appwrite account using `appwrite login`
3. initialize your project with `appwrite init project` and follow steps in wizard
4. pull your existing bucket using `appwrite pull buckets`
5. initialize an existing/new project with `appwrite init project`, but make sure it's different from the pre-existing project in `appwrite.json`
6. push your bucket in your `appwrite.json` using `appwrite push buckets`. This will automatically create a bucket that matches the settings of your bucket in your `appwrite.json`